src/components/Camera/hooks/usePrepareStoreAndNavigate.ts

```diff
@@ -25,6 +25,7 @@ const usePrepareStoreAndNavigate = ( ): Function => {
   const setSavingPhoto = useStore( state => state.setSavingPhoto );
   const setCameraState = useStore( state => state.setCameraState );
   const setSentinelFileName = useStore( state => state.setSentinelFileName );
+  const isSuggestionsFlowMode = useStore( state => state.layout.isSuggestionsFlowMode );
```

Comment:
could we add "advanced" to the naming here? maybe something like `isAdvancedSuggestionsMode`? i'd love to remember that this is a power user setting when i look at this code, and right now i'd be kind of confused by the naming about why we need this mode at all

Reply:
Renamed